src/main/index.ts

```diff
@@ -76,7 +79,7 @@ app.on("ready", async () => {
   }
 
   LensExtensionsApi.windowManager = windowManager = new WindowManager(proxyPort);
-  extensionLoader.init(); // call after windowManager to see splash earlier
+  extensionLoader.init(await extensionManager.load()); // call after windowManager to see splash earlier
```

Comment:
why are we `load`-ing the extrensionManager twice in the same function?

Reply:
wat? 🥱

Reply:
oh oops sorry, I misread `extensionStore` as `extensionManager`. My bad.